src/templates/index.html

```diff
@@ -118,7 +181,155 @@ <h2>Support & Additional Resources</h2>
             <p>For more information and detailed usage instructions, refer to our <a href="https://github.com/DSSGxMunich/dssgx_fire_hydrant_range_finder">GitHub repository</a>.</p>
         </div>
     </div>
+    
 </div>
+<script type="text/javascript">
+    function UploadFile() {
+        //Reference the FileUpload element.
+        var fileUpload = document.getElementById("fileUpload");
+ 
+        //Validate whether File is valid Excel file.
+        var regex = /^([a-zA-Z0-9\s_\\.\-:])+(.xls|.xlsx)$/;
```

Comment:
Seems like this regrex expression is the cause of the error in uploading my template. It does not allow parentheses in the file name. This isn't obvious to the user, so instead try

/^([a-zA-Z0-9\s_().\\:\-])+(.xls|.xlsx)$/

and test with a filename that contains "(" and ")"

Reply:
The suggested regex was'nt working for some reason.
Instead I ve changed the regex to /^([a-zA-Z0-9\s_()\\.\-:])+(.xls|.xlsx)$/ and it seems working for the test case. Attached the test image for reference.
![image](https://github.com/DSSGxMunich/hydroxplorer-web-app/assets/60874029/4bc9e361-7806-477d-bf8a-5cea452b8a94)